frontend/src/components/IC50Line/IC50Tooltip.tsx

```diff
@@ -21,7 +21,7 @@ export function IC50Tooltip({ ic50, className }: IC50TooltipProps): React.ReactE
   const errTooltip = (msg: string) => {
     return (
       <Tooltip text={msg} type="error">
-        <span className={cx("ic50value__none", className)} />
+        <span className={cx("ic50value__none", className)}>error</span>
```

Comment:
There was `-` in design for this case, but apparently it was lost. I've asked to update it, you can see it now
![image](https://user-images.githubusercontent.com/2556409/117718015-3680c280-b1e4-11eb-8e08-cbef40f32f3a.png)

It's in "Default dashboard+tooltips". But if we change it to `-`, there is a problem because we show tooltip only when cursor is above `-` which is quite small. If you have an idea what to do with it (e. g. show tooltip when cursor is above the whole area where text (numbers) is usually written), let's change `error` to `-` and do it. Otherwise let's just skip this problem and write `error` which is larger.

Reply:
So, as for me `error` is better, because `-` is smth strange for me. I suggest to leave it. As for experiment `-` we don't have it now, will add